### ISSUE TITLE
The EdgeTX 2.7.1 screen flickers when switching from iNAV to another widget.

### ISSUE THREAD

**Issue by AJM-DSC** [EdgeTX/edgetx#2184]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

The EdgeTX 2.7.1 screen flickers when switching from iNAV to another widget. The "showall 0.9" widget screen has a flickering bar on the right side of the screen when I change the view from iNAV (full screen) to showall (full screen). In version 2.8, I did not notice this effect. Switching from a screen other than iNAV does not produce this phenomenon. I will try to enclose a video illustrating the problem.
I didn't notice this effect in version 2.8 (dev).

### Expected Behavior

Removal of this flickering effect.

### Steps To Reproduce

1. install and configure iNAV, showall 0.9 and one other widget.
2. set iNAV screen visible.
3. change screen to showall.
4. such effect is visible when you turn from iNAV to showall full screen wiev.

### Version

2.7.1

### Transmitter

Radiomaster TX16S

**Comment by gagarinlg** [comment]
As there is probably no another 2.7 release and it does not happen on 2.8, I am inclined to close this issue.
Are there any other opinions?

**Comment by AJM-DSC** [comment]
Hi,

 

I observed this phenomenon on version 2.7.1 when switching from the iNAV to the ‘showall 0.9’ screen. 

I think the problem will disappear with version 2.8. 

 

BTW: when are you planning to release version 2.8 officially? In the last edition of "nighty", 

I still see the problems I reported in another thread (@ in place of the "deg" symbol in GPS coordinates, 

delay or failure to update the display of the 6-position switch status).

 

Regards,

 

Andrzej Majewski
